### DISCUSSION TITLE
mac 歌曲无法添加播放

### DISCUSSION THREAD

**Discussion by lqw1** [1715173329/UnblockNeteaseMusic#822] [Q&A]
版本号 2.3.9
<img width="574" alt="image" src="https://user-images.githubusercontent.com/28446076/183896471-a10345a5-c694-46f1-819e-bb448c755bff.png">

启动命令 node app.js -p 80:443 -f 115.236.121.4

不是灰色了 但是还是无法播放

**Comment by 1715173329** [comment]
请设定 endpoint，详细参见 readme 和其他 issue。
